### ISSUE TITLE
Keyboard users cannot interact with map or graph

### ISSUE THREAD

**Issue by SteveChab** [cmu-delphi/www-covidcast#82]
Clicking on the map reveals information in tooltips and changes the graph. Keyboard only users cannot access/interact with this information.
